addons/mail/static/src/components/attachment_delete_confirm_dialog/attachment_delete_confirm_dialog.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<templates xml:space="preserve">
+    <t t-name="mail.AttachmentDeleteConfirmDialog" owl="1">
+        <Dialog contentClass="'o_AttachmentDeleteConfirmDialog'" title="CONFIRMATION" size="'medium'" t-ref="dialog">
+            <p class="o_AttachmentDeleteConfirmDialog_mainText" t-esc="getText()"/>
+            <t t-if="attachment.messages.length > 0">
+                <p class="o_AttachmentDeleteConfirmDialog_secondaryText">
+                    This will also delete messages containing this attachment if they are now empty.
```

Comment:
This text feels strange. Can't we just filter the message that indeed would be empty, and show only the text if actually relevant? Conditional text is very confusing.

Honestly it feels useless to have in the first place, it's obvious by itself that such messages will be deleted. It's not like there will be data lost or anything, the user doesn't need to be warned about this on top of being warned about the deletion of the attachment itself.

Reply:
I'm ok without message, @alexkuhn asked for it so maybe he has arguments to continue that way ?

Reply:
Let's ignore that aspect. Keep 13.4 fixes for the actual fixes, and do improvements in master.

Motivation for prompt was to remove (or at least, reduce) confusion on user of "flicker" when deleting the attachment, due to message being also deleted, which is not super obvious that such a clean would do that (ofc it makes sense, but confusing and scary when you are not fully knowledgeable).

In master, we'll take time to find appropriate UX for such things, be it with prompt/smoother animation/user actions/etc. It's totally not worth finding and implementing solution right now, just being aware of the problem is good.